src/components/Popup/Popup.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './Popup.css';
+
+function Popup(props) {
+
+    const { isOpened, onClose } = props;
+
+    return (
+        <div className={`popup ${isOpened && 'popup_opened'}`}>
+            <div className='popup__container'>
+                <button className='popup__button-close' type='button' onClick={onClose}>&times;</button>
+                {props.children}
```

Comment:
Just to be consistent:
```suggestion
    const { isOpened, onClose, children } = props;

    return (
        <div className={`popup ${isOpened && 'popup_opened'}`}>
            <div className='popup__container'>
                <button className='popup__button-close' type='button' onClick={onClose}>&times;</button>
                {children}
```